anesthetic/samples.py

```diff
@@ -1138,6 +1140,57 @@ def live_points(self, logL=None):
         i = ((self.logL >= logL) & (self.logL_birth < logL)).to_numpy()
```

Comment:
Currently tests are failing due to the second check here, is this defined correctly (should the logL_birth check be inclusive)?

Reply:
It all depends on the convention as to whether or not you generate new live points subject to a strict or non-strict hard likelihood constraint. At the moment many nested samplers specify strict (polychord and multinest for example), so logL_birth < logL, but this is not necessarily required. Perhaps we could create an issue so that anesthetic can work with either convention?

Reply:
I think that the convention is consistent i.e. this version of the strictness ensures that the 'live points at iteration i' corresponds to 'live points at loglikelihood logL' in a sensible way.

I think the real issue is that samples.live_points() (i.e. the None functionality) is not returning the right thing. This should return 'the last set of live points', and at the moment returns the penultimate set.

I've pushed a new function 'contour' that aims to achieve this. Could you tidy up the docstrings with a new users eye?